test/index.test.ts

```diff
@@ -7,7 +7,7 @@ import { KubescapeApi, KubescapeUi, IKubescapeConfig } from '../src/index'
 const seconds = (n : number) => n * 1000
 const minutes = (n : number) => n * 1000 * 60 
 
-const DEFAULT_KUBESCAPE_VERSION = "v2.0.176";
+const DEFAULT_KUBESCAPE_VERSION = "v2.3.0";
```

Comment:
I don't know if this will break the vscode and lens extensions, but since the arm64 binaries as well as kubescape.exe are introduced in v2.3.0, maybe we can postpone this merge and wait for the vscode and lens extensions to get adapted to the latest version.

Reply:
Looks like here we just specify the kubescape version for testing. This is not related to the vscode and lens extensions. We shall wait for vscode and lens extensions' kubescape upgrade to version greater than v2.3.0